UnityProject/Assets/Scripts/UI/Objects/Atmospherics/Canister/NumberSpinner.cs

```diff
@@ -134,10 +134,12 @@ public void DisplaySpinTo(int newValue)
 
 			if (newValue > MaxValue || newValue < 0)
 			{
-				Logger.LogErrorFormat("New value {0} is out of range, should be between 0 and {1} inclusive",
+				Logger.LogWarningFormat("New value {0} is out of range, should be between 0 and {1} inclusive",
 					Category.Atmos, newValue, MaxValue);
 			}
 
+			newValue = Mathf.Clamp(newValue, MaxValue, 0);
```

Comment:
Is this reversed? Clamp should take in a (value, minimum, maximum) but this looks like its doing it in reverse with the max value